auth/openid_connect.adoc

```diff
@@ -242,14 +266,24 @@ To configure the appliance for OpenID-Connect Authentication:
                         --oidc-url
                         --oidc-client-id
                         --oidc-client-secret
+                        [--oidc-introspection-endpoint]
                         [--oidc-enable-sso]
+                        [--oidc-insecure]
 ```
 
+When the `--oidc-introspection-endpoint` is not specified, an attempt will be made to
+fetch the introspection endpoint by querying the OpenID-Connect Provider URL.
+
 When the `--oidc-client-host` is not specified, the configured appliance host
 is used for the ServerName and OpenID-Connect Redirect URI in the
 OpenID-Connect Apache configuration file. The `--oidc-client-host` is
 usually the externally reachable FQDN of the appliance.
 
+When the `--oidc-insecure` is specified not SSL verification will be done when
```

Comment:
not -> no

Reply:
Know not what not I know. Thank you for untying this knot. Oh no! 😉